docs/topics/gradle.md

```diff
@@ -133,6 +133,158 @@ sourceSets {
 </tab>
 </tabs>
 
+### Check for JVM target compatibility of related compile tasks
+
+In the build module, you may have related compile tasks, for example:
+* `compileKotlin` and `compileJava`
+* `compileTestKotlin` and `compileTestJava`
+
+> `main` and `test` source set compile tasks are not related.
+>
+{type="note"}
+
+For such related tasks, the Kotlin Gradle plugin checks for JVM target compatibility. Different values of `jvmTarget` in the `kotlin` extension
+and [`targetCompatibility`](https://docs.gradle.org/current/userguide/java_plugin.html#sec:java-extension)
+in the `java` extension cause incompatibility. For example:
+the `compileKotlin` task has `jvmTarget=1.8`, and
+the `compileJava` task has (or [inherits](https://docs.gradle.org/current/userguide/java_plugin.html#sec:java-extension)) `targetCompatibility=15`.
+
+Control the behavior of this check by setting the `kotlin.jvm.target.validation.mode` property in the `build.gradle` file equal to:
+* `warning` – the default value; the Kotlin Gradle plugin will print a warning message.
+* `error` – the plugin will fail the build.
+* `ignore` – the plugin will skip the check and won't produce any messages.
+
+### Set custom JDK home
+
+> [Setting `jdkHome` option directly is deprecated](https://youtrack.jetbrains.com/issue/KT-46541).
+>
+{type="warning"}
+
+By default, Kotlin compile tasks use the current Gradle JDK. However, if you need, you can set the JDK home in the following ways:
```

Comment:
Is this ok as the description why to set custom JDK home?

Reply:
The edits are in this [commit](https://github.com/JetBrains/kotlin-web-site/pull/2643/commits/9ddff33d6ac6a98e7c0dd46a764d372966a59840).